e2e-environment/run-e2e-tests.sh

```diff
@@ -27,6 +27,8 @@ docker run --detach --rm \
   raidennetwork/lightclient-e2e-environment \
   >/dev/null
 
+sleep 5
```

Comment:
Why is this needed?

Reply:
On local runs, I've seen the LCs starting too quickly after the container got started, and failed because it'd not be able to hit matrix/synapse in a usable state. Giving some time for the containers to start did fix it for me.